src/app/pages/home/profile/profile.page.ts

```diff
@@ -41,63 +30,46 @@ export class ProfilePage implements OnInit {
     targetHeight: 374
   }
 
-  constructor(private authService: AuthService, private userService: UserService, private cardService: CardService, private activatedRoute: ActivatedRoute,
-                private router: Router, private camera: Camera, public actionSheetController: ActionSheetController, public alertController: AlertController, private toastCtrl: ToastController) {
-    console.log("Profile page started (constructor)");
-
-    console.log('Is authenticated:', this.authService.isAuthenticated());
-
-    //this.id = this.cardService.getCard();
```

Comment:
Been literally resisting the urge to delete this. Glad it got your seal of removal lolol

Reply:
Some stuff may need to be removed later. I just needed to make it neat to keep my head from exploding. It's just for readability, not for approval.